course_catalog/urls.py

```diff
@@ -72,15 +67,4 @@
         r"^api/v0/ocw-course-report", views.ocw_course_report, name="ocw-course-report"
     ),
     re_path(r"^podcasts/rss_feed", views.podcast_rss_feed, name="podcast-rss-feed"),
-    re_path(r"api/v0/schema/$", SpectacularAPIView.as_view(), name="schema"),
-    re_path(
-        "api/v0/schema/swagger-ui/",
-        SpectacularSwaggerView.as_view(url_name="schema"),
-        name="swagger-ui",
-    ),
-    re_path(
-        "api/v0/schema/redoc/",
-        SpectacularRedocView.as_view(url_name="schema"),
-        name="redoc",
-    ),
```

Comment:
Which docs get served is controlled via the `SERVE_URLCONF` env variable. As far as I know, we can't serve the old catalog docs at one URL and the new learning resource docs at another, at least not with the builtin `drf-spectacular` views.